content/general/queues.textile

```diff
@@ -94,7 +94,7 @@ Whilst the queue dashboard stats show the current state of your queue, your app
 
 h3(#testing-rules). Testing your queue rules
 
-Once your "Ably Queue":https://ably.com/integrations is provisioned, and your "Ably Queue rules":https://faqs.ably.com/how-can-i-set-up-a-queue-rule are configured, there are a number of ways we recommend customers can debug the configured rules and queues:
+Once your "Ably Queue":/general/queue is provisioned, and your "Ably Queue rules":https://faqs.ably.com/how-can-i-set-up-a-queue-rule are configured, there are a number of ways we recommend customers can debug the configured rules and queues:
```

Comment:
I believe this should be @/general/queues@

Reply:
Well spotted! Should now be fixed: https://github.com/ably/docs/pull/1510/commits/4336cc0b0e93bc80551f2e0d1832b31fbf22caf1